### DISCUSSION TITLE
Change max Memory of mysql from dashboard

### DISCUSSION THREAD

**Discussion by drupaladmin** [Releem/releem-agent#208] [Feature Requests]
Will be helpful if i can change max Memory of mysql from dashboard ( actually i can set this parameter only on setup ), for example i have likes 5-6 server with out Memory limite at mysql and want to resize the Memory limit without uninstall/ reinstall releem.

**Comment by sujoydhar2k20** [comment]
Mostly needed this one

**Comment by drupaladmin** [comment]
We've improved Server Settings and now you could change MySQL Memory Limit from the Releem Dashboard.

![Roman Agabekov tweet image](https://github.com/Releem/mysqlconfigurer/assets/4052657/8eedb626-8da1-4d93-9329-012f931430b3)
